Gemfile.lock

```diff
@@ -1,40 +1,40 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.2.1)
-      actionview (= 6.0.2.1)
-      activesupport (= 6.0.2.1)
+    actionpack (6.0.2.2)
+      actionview (= 6.0.2.2)
+      activesupport (= 6.0.2.2)
       rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.2.1)
-      activesupport (= 6.0.2.1)
+    actionview (6.0.2.2)
+      activesupport (= 6.0.2.2)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.2.1)
-      activesupport (= 6.0.2.1)
-    activerecord (6.0.2.1)
-      activemodel (= 6.0.2.1)
-      activesupport (= 6.0.2.1)
-    activesupport (6.0.2.1)
+    activemodel (6.0.2.2)
+      activesupport (= 6.0.2.2)
+    activerecord (6.0.2.2)
+      activemodel (= 6.0.2.2)
+      activesupport (= 6.0.2.2)
+    activesupport (6.0.2.2)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
       zeitwerk (~> 2.2)
     builder (3.2.4)
-    concurrent-ruby (1.1.5)
-    crass (1.0.5)
+    concurrent-ruby (1.1.6)
+    crass (1.0.6)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
     erubi (1.9.0)
-    i18n (1.7.0)
+    i18n (1.8.2)
```

Comment:
### [i18n](https://github.com/ruby-i18n/i18n)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.7.0 → 1.8.2

[_[change-log](https://github.com/svenfuchs/i18n/releases), [source-code](https://github.com/svenfuchs/i18n)_]





<details>
<summary>Commits</summary>

A change of **42** commits. See the full changes on [the compare page](https://github.com/ruby-i18n/i18n/compare/v1.7.0...v1.8.2).

These are the first 10 commits:
- (dcda31c) [Exclude :count option on retrieve link](https://github.com/ruby-i18n/i18n/commit/dcda31cc1bc367496cfa52f706b5e8c7b7fd26ff)
- (97c8a53) [Update ruby.yml](https://github.com/ruby-i18n/i18n/commit/97c8a53981f23d4bfdcf9f940974fc64a56d4178)
- (18e91e3) [Update ruby.yml](https://github.com/ruby-i18n/i18n/commit/18e91e3e3f2f88b8b588ecd7bcfea63ac0c39407)
- (e64c5c2) [Exclude Ruby 2.4.x + Rails master Gemfile build](https://github.com/ruby-i18n/i18n/commit/e64c5c29c4076b18083419f3d539798430273fd9)
- (b1148ca) [One i in gemfile](https://github.com/ruby-i18n/i18n/commit/b1148caf642bc0ff334220a0966f3100b7699b60)
- (8684c89) [Add Ruby 2.3 to Ruby pipeline](https://github.com/ruby-i18n/i18n/commit/8684c89eed68e4c3c28ed9592502397c5a3cfde0)
- (d5f575d) [Add JRuby to build pipeline](https://github.com/ruby-i18n/i18n/commit/d5f575d180bba1e922884918490d6c7c43357e95)
- (04a814b) [Merge pull request #480 from Tietew/exclude-count-on-retriev…](https://github.com/ruby-i18n/i18n/commit/04a814bab173d495c44fe683309aaff18a42593e)
- (af76109) [Add tests for existing behavior](https://github.com/ruby-i18n/i18n/commit/af76109eb1bbfb862c72aab54f5dedd785b0f170)
- (06aa9ac) [Merge pull request #495 from ghiculescu/pluralization_fallba…](https://github.com/ruby-i18n/i18n/commit/06aa9ac0685aa74eca8cdb4cd7eb10c75969eb6d)


</details>